recipes/easydict/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "1.9" %}
+package:
+    name: easydict
+    version: {{ version }}
+source:
+    sha256: 3f3f0dab07c299f0f4df032db1f388d985bb57fa4c5be30acd25c5f9a516883b
+    url: https://pypi.io/packages/source/e/easydict/easydict-{{ version }}.tar.gz
+build:
+    number: '0'
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+    host:
+        - python
+        - pip
+    run:
+        - python
+test:
+    imports:
+        - easydict
+about:
+    description: "*EasyDict* allows to access dict values as attributes (works recursively).\
```

Comment:
You should be able to use YAML `|`-syntax to clean this up and remove a lot of the escaping, line continuation, and newline characters.

Reply:
Thank you very much.